ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__arm64-nightly.yaml

```diff
@@ -198,6 +204,56 @@ tests:
     test:
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: azure-ipi-ovn-hypershift-guest-f7
```

Comment:
According to the team agreement, if a test support both amd64 and arm64, then we move the test to multi. I'm just not sure if this rule applies to hypershift.

/cc @jianlinliu

Reply:
@liangxia the agreement is still available for HyperShift. What this PR doing is for Multi-Arch, ARM for management cluster and X86 for hosted cluster. The agreement will be available if HyperShift need to make hosted cluster as heterogeneous.

Reply:
Just to add some clarification - only a multi arch release image is needed for a HostedCluster to have heterogenous NodePools. The arch of the mgmt cluster doesn't particularly matter.